packages/upswyng-server/src/routes/api/weather.ts

```diff
@@ -0,0 +1,113 @@
+import axios from "axios";
+
+const getCurrentWeather = async (
+  lat: number | null,
```

Comment:
super personal technique is to not abbreviate "latitude" and "longitude" because then we get a mix of `lng` `long` `lon`. For the pass-through type from the openweather API I would not modify.